app/Main.hs

```diff
@@ -121,6 +121,9 @@ main = do
       putStrLn (Yaml.prettyPrintParseException err)
       exitFailure
     Right C.Config {..} -> do
+      unless (configTargetSystem == Windows) $ do
+        putStrLn "Windows is not supported as a target platform for deployment"
```

Comment:
Do you think we can remove Windows from `TargetSystem` since currently we don't support that platform?